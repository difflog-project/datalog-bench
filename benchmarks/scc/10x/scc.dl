.type N
.decl edge(v0: N, v1: N)
.input edge

.decl path(v0: N, v1: N)

.decl scc(v0: N, v1: N)
.output scc

path(x, y) :- edge(x, y).
path(x, z) :- path(x, y), edge(y, z).
scc(x, y) :- path(x, y), path(y, x).
