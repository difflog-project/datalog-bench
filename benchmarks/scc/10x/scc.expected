v0_5	v0_5
v0_5	v0_3
v0_5	v0_6
v0_5	v0_7
v0_5	v0_4
v0_3	v0_5
v0_3	v0_3
v0_3	v0_6
v0_3	v0_7
v0_3	v0_4
v0_6	v0_5
v0_6	v0_3
v0_6	v0_6
v0_6	v0_7
v0_6	v0_4
v0_7	v0_5
v0_7	v0_3
v0_7	v0_6
v0_7	v0_7
v0_7	v0_4
v0_4	v0_5
v0_4	v0_3
v0_4	v0_6
v0_4	v0_7
v0_4	v0_4
v1_5	v1_5
v1_5	v1_3
v1_5	v1_6
v1_5	v1_7
v1_5	v1_4
v1_3	v1_5
v1_3	v1_3
v1_3	v1_6
v1_3	v1_7
v1_3	v1_4
v1_6	v1_5
v1_6	v1_3
v1_6	v1_6
v1_6	v1_7
v1_6	v1_4
v1_7	v1_5
v1_7	v1_3
v1_7	v1_6
v1_7	v1_7
v1_7	v1_4
v1_4	v1_5
v1_4	v1_3
v1_4	v1_6
v1_4	v1_7
v1_4	v1_4
v2_5	v2_5
v2_5	v2_3
v2_5	v2_6
v2_5	v2_7
v2_5	v2_4
v2_3	v2_5
v2_3	v2_3
v2_3	v2_6
v2_3	v2_7
v2_3	v2_4
v2_6	v2_5
v2_6	v2_3
v2_6	v2_6
v2_6	v2_7
v2_6	v2_4
v2_7	v2_5
v2_7	v2_3
v2_7	v2_6
v2_7	v2_7
v2_7	v2_4
v2_4	v2_5
v2_4	v2_3
v2_4	v2_6
v2_4	v2_7
v2_4	v2_4
v3_5	v3_5
v3_5	v3_3
v3_5	v3_6
v3_5	v3_7
v3_5	v3_4
v3_3	v3_5
v3_3	v3_3
v3_3	v3_6
v3_3	v3_7
v3_3	v3_4
v3_6	v3_5
v3_6	v3_3
v3_6	v3_6
v3_6	v3_7
v3_6	v3_4
v3_7	v3_5
v3_7	v3_3
v3_7	v3_6
v3_7	v3_7
v3_7	v3_4
v3_4	v3_5
v3_4	v3_3
v3_4	v3_6
v3_4	v3_7
v3_4	v3_4
v4_5	v4_5
v4_5	v4_3
v4_5	v4_6
v4_5	v4_7
v4_5	v4_4
v4_3	v4_5
v4_3	v4_3
v4_3	v4_6
v4_3	v4_7
v4_3	v4_4
v4_6	v4_5
v4_6	v4_3
v4_6	v4_6
v4_6	v4_7
v4_6	v4_4
v4_7	v4_5
v4_7	v4_3
v4_7	v4_6
v4_7	v4_7
v4_7	v4_4
v4_4	v4_5
v4_4	v4_3
v4_4	v4_6
v4_4	v4_7
v4_4	v4_4
v5_5	v5_5
v5_5	v5_3
v5_5	v5_6
v5_5	v5_7
v5_5	v5_4
v5_3	v5_5
v5_3	v5_3
v5_3	v5_6
v5_3	v5_7
v5_3	v5_4
v5_6	v5_5
v5_6	v5_3
v5_6	v5_6
v5_6	v5_7
v5_6	v5_4
v5_7	v5_5
v5_7	v5_3
v5_7	v5_6
v5_7	v5_7
v5_7	v5_4
v5_4	v5_5
v5_4	v5_3
v5_4	v5_6
v5_4	v5_7
v5_4	v5_4
v6_5	v6_5
v6_5	v6_3
v6_5	v6_6
v6_5	v6_7
v6_5	v6_4
v6_3	v6_5
v6_3	v6_3
v6_3	v6_6
v6_3	v6_7
v6_3	v6_4
v6_6	v6_5
v6_6	v6_3
v6_6	v6_6
v6_6	v6_7
v6_6	v6_4
v6_7	v6_5
v6_7	v6_3
v6_7	v6_6
v6_7	v6_7
v6_7	v6_4
v6_4	v6_5
v6_4	v6_3
v6_4	v6_6
v6_4	v6_7
v6_4	v6_4
v7_5	v7_5
v7_5	v7_3
v7_5	v7_6
v7_5	v7_7
v7_5	v7_4
v7_3	v7_5
v7_3	v7_3
v7_3	v7_6
v7_3	v7_7
v7_3	v7_4
v7_6	v7_5
v7_6	v7_3
v7_6	v7_6
v7_6	v7_7
v7_6	v7_4
v7_7	v7_5
v7_7	v7_3
v7_7	v7_6
v7_7	v7_7
v7_7	v7_4
v7_4	v7_5
v7_4	v7_3
v7_4	v7_6
v7_4	v7_7
v7_4	v7_4
v8_5	v8_5
v8_5	v8_3
v8_5	v8_6
v8_5	v8_7
v8_5	v8_4
v8_3	v8_5
v8_3	v8_3
v8_3	v8_6
v8_3	v8_7
v8_3	v8_4
v8_6	v8_5
v8_6	v8_3
v8_6	v8_6
v8_6	v8_7
v8_6	v8_4
v8_7	v8_5
v8_7	v8_3
v8_7	v8_6
v8_7	v8_7
v8_7	v8_4
v8_4	v8_5
v8_4	v8_3
v8_4	v8_6
v8_4	v8_7
v8_4	v8_4
v9_5	v9_5
v9_5	v9_3
v9_5	v9_6
v9_5	v9_7
v9_5	v9_4
v9_3	v9_5
v9_3	v9_3
v9_3	v9_6
v9_3	v9_7
v9_3	v9_4
v9_6	v9_5
v9_6	v9_3
v9_6	v9_6
v9_6	v9_7
v9_6	v9_4
v9_7	v9_5
v9_7	v9_3
v9_7	v9_6
v9_7	v9_7
v9_7	v9_4
v9_4	v9_5
v9_4	v9_3
v9_4	v9_6
v9_4	v9_7
v9_4	v9_4
